src/language/utils.ts

```diff
@@ -14,111 +14,111 @@
+/**
+    * @returns true if any modifier kinds passed along exist in the given modifiers array
+    */
+export function hasModifier(modifiers: ts.ModifiersArray, ...modifierKinds: ts.SyntaxKind[]) {
+    if (modifiers == null || modifierKinds == null) {
+        return false;
     }
 
-    /**
-     * Determines if the appropriate bit in the parent (VariableDeclarationList) is set,
-     * which indicates this is a "let" or "const".
-     */
-    export function isBlockScopedVariable(node: ts.VariableDeclaration | ts.VariableStatement): boolean {
-        const parentNode = (node.kind === ts.SyntaxKind.VariableDeclaration)
-            ? (<ts.VariableDeclaration> node).parent
-            : (<ts.VariableStatement> node).declarationList;
+    return modifiers.some((m) => {
+        return modifierKinds.some((k) => m.kind === k);
+    });
+}
 
-        return isNodeFlagSet(parentNode, ts.NodeFlags.Let)
-            || isNodeFlagSet(parentNode, ts.NodeFlags.Const);
-    }
+/**
+    * Determines if the appropriate bit in the parent (VariableDeclarationList) is set,
+    * which indicates this is a "let" or "const".
+    */
+export function isBlockScopedVariable(node: ts.VariableDeclaration | ts.VariableStatement): boolean {
+    const parentNode = (node.kind === ts.SyntaxKind.VariableDeclaration)
+        ? (<ts.VariableDeclaration> node).parent
+        : (<ts.VariableStatement> node).declarationList;
 
-    export function isBlockScopedBindingElement(node: ts.BindingElement): boolean {
-        let currentParent = node.parent;
-        while (currentParent.kind !== ts.SyntaxKind.VariableDeclaration) {
-            if (currentParent.parent == null) {
-                // if we didn't encounter a VariableDeclaration, this must be a function parameter, which is block scoped
-                return true;
-            } else {
-                currentParent = currentParent.parent;
-            }
+    return isNodeFlagSet(parentNode, ts.NodeFlags.Let)
+        || isNodeFlagSet(parentNode, ts.NodeFlags.Const);
+}
+
+export function isBlockScopedBindingElement(node: ts.BindingElement): boolean {
+    let currentParent = node.parent;
+    while (currentParent.kind !== ts.SyntaxKind.VariableDeclaration) {
+        if (currentParent.parent == null) {
+            // if we didn't encounter a VariableDeclaration, this must be a function parameter, which is block scoped
+            return true;
+        } else {
+            currentParent = currentParent.parent;
         }
-        return isBlockScopedVariable(<ts.VariableDeclaration> currentParent);
     }
+    return isBlockScopedVariable(<ts.VariableDeclaration> currentParent);
+}
 
-    /**
-     * Bitwise check for node flags.
-     */
-    export function isNodeFlagSet(node: ts.Node, flagToCheck: ts.NodeFlags): boolean {
-        /* tslint:disable:no-bitwise */
-        return (node.flags & flagToCheck) !== 0;
-        /* tslint:enable:no-bitwise */
-    }
+/**
+    * Bitwise check for node flags.
+    */
```

Comment:
@weswigham - agree, `tsfmt` doesn't need to be in this PR. I'll pick up on comments in the PR you created for it.